pkg/controller/policyrecommendation/policyrecommendation_controller.go

```diff
@@ -394,7 +410,10 @@ func (r *ReconcilePolicyRecommendation) Reconcile(ctx context.Context, request r
 		rcertificatemanagement.CertificateManagement(&rcertificatemanagement.Config{
```

Comment:
I think we only need to create the certificate only for management and standalone, and not for all clusters.